src/audio-worklet/design-pattern/lib/free-queue.js

```diff
@@ -25,6 +25,17 @@ const MAX_CHANNEL_COUNT = 32;
 // WebAudio's render quantum size.
 const RENDER_QUANTUM_FRAMES = 128;
 
+/**
+ * A shared storage for FreeQueue operation backed by SharedArrayBuffer.
```

Comment:
I don't see the matching implementation of this comment. Where is it?

Reply:
The code related to the old `free-queue` class starts from line `317`.